apps/ui/src/components/FormSpaceProfile.vue

```diff
@@ -19,70 +31,129 @@ const emit = defineEmits<{
   (e: 'pick', field: any);
 }>();
 
-const definition = computed(() => {
+const isOffchainNetwork = computed(
+  () => props.space && offchainNetworks.includes(props.space.network)
+);
+
+const profileDefinition = computed(() => {
+  const offchainProperties = {
+    categories: {
+      type: 'array',
+      items: {
+        type: 'string',
+        enum: SPACE_CATEGORIES.map(c => c.id)
+      },
+      options: SPACE_CATEGORIES,
+      title: 'Categories',
+      examples: ['Select up to 2 categories'],
+      maxItems: 2
+    }
+  };
+
   return {
     type: 'object',
-    title: 'Space',
+    title: 'Profile',
     additionalProperties: true,
     required: ['name'],
     properties: {
-      avatar: {
-        type: 'string',
-        format: 'stamp',
-        title: 'Avatar',
-        default:
-          props.id ||
-          '0x2121212121212121212121212121212121212121212121212121212121212121'
-      },
       name: {
         type: 'string',
         title: 'Name',
         minLength: 1,
+        maxLength: 32,
         examples: ['Space name']
       },
       description: {
         type: 'string',
         format: 'long',
         title: 'About',
+        maxLength: 160,
         examples: ['Space description']
       },
+      ...(isOffchainNetwork.value ? offchainProperties : {})
+    }
+  };
+});
+
+const votingPowerDefinition = computed(() => ({
+  type: 'object',
+  title: 'Voting power',
+  additionalProperties: true,
+  required: [],
+  properties: {
+    symbol: {
+      type: 'string',
+      title: 'Voting power symbol',
+      examples: ['e.g. VP'],
+      maxLength: isOffchainNetwork ? 16 : MAX_SYMBOL_LENGTH,
+      minLength: isOffchainNetwork ? 1 : undefined
+    }
+  }
+}));
+
+const socialAccountsDefinition = computed(() => {
+  const onchainProperties = {
+    discord: {
+      type: 'string',
+      format: 'discord-handle',
+      title: 'Discord',
+      examples: ['Discord handle or invite code']
+    }
+  };
+
+  const offchainProperties = {
+    coingecko: {
+      type: 'string',
+      format: 'coingecko-handle',
+      title: 'Coingecko',
+      examples: ['CoinGecko handle'],
```

Comment:
With official casing and example
```suggestion
      title: 'CoinGecko handle',
      examples: ['e.g. uniswap'],
```